### PR TITLE
feat(i18n): AppUpgradeModal,CloudStorageContainer

### PR DESCRIPTION
**Pull request by Leooeloel**

### CODE REVIEW

File: packages/flat-i18n/locales/zh-CN.json [hunk 1]
```diff
@@ -133,5 +133,38 @@
...
+  "modify": "修改",
+  "modify-this-room": "修改本次房间",
+  "make-sure-to-modify-this-room": "确定修改本次房间？",
+  "make-sure-to-modify-room": "确定修改房间？",
+  "make-sure-to-modify-the-series-of-periodic-rooms": "确定修改该系列周期性房间？",
+  "back-tips": "房间信息未保存，确定返回？",
+  "begin-time-cannot-be-in-the-past": "开始时间不能为过去",
+  "end-time": "结束时间",
+  "frequency": "重复频率",
+  "end-series": "结束重复",
```
Comment (Cheerego7) [inline packages/flat-i18n/locales/zh-CN.json]: "end-repeat"

File: packages/flat-i18n/locales/en.json [hunk 2]
```diff
@@ -133,5 +133,38 @@
   "enter-room": "Enter",
   "modify-room": "Modify",
   "replication-success": "Copy successfully",
-  "view-replay": "View replay"
+  "view-replay": "View replay",
+  "back-tips": "The room information is not saved, are you sure to back?",
+  "begin-time-cannot-be-in-the-past": "Start time cannot be in the past",
+  "schedule": "Schedule",
+  "end-series": "End series",
```
Comment (Cheerego7) [inline packages/flat-i18n/locales/en.json]: "end-repeat": "End Repeat"

File: packages/flat-i18n/locales/zh-CN.json [hunk 3]
```diff
@@ -133,5 +133,38 @@
...
+  "room-duration-limit": "房间时长最少 {{minutes}} 分钟",
+  "used-storage": "已使用 {{usage}}",
+  "delete": "删除",
+  "upload": "上传",
+  "my-cloud ": "我的云盘",
+  "version-updates": "版本更新",
+  "downloading": "下载中",
+  "update-failed-tips": "更新失败，请重新打开程序",
+  "update-now": "立即更新",
+  "new-version-tips": "发现新版本{{version}}, 请更新到最新版本获取更好的产品体验"
```
Comment (BlackHole1) [inline packages/flat-i18n/locales/zh-CN.json]:
```suggestion
  "new-version-tips": "发现新版本: {{version}}, 请更新到最新版本获取更好的产品体验"
```

File: packages/flat-i18n/locales/en.json [hunk 4]
```diff
@@ -133,5 +133,38 @@
...
+  "roomtype": "Room type: ",
+  "theme": "Theme",
+  "theme-can-be-up-to-50-characters": "Subject is up to 50 characters",
+  "type": "Type",
+  "delete": "Delete",
+  "my-cloud ": "My cloud disk",
+  "upload": "Upload",
+  "used-storage": "Used {{usage}}",
+  "downloading": "Downloading",
+  "new-version-tips": "Found a new version {{version}}, please update to the latest version for a better product experience",
```
Comment (BlackHole1) [inline packages/flat-i18n/locales/en.json]:
```suggestion
  "new-version-tips": "Found a new version: {{version}}, please update to the latest version for a better product experience",
```

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Review by BlackHole1 — APPROVED** [review]

**Review by Cheerego7 — APPROVED** [review]

**Pull request merged**